pkg/generate/app/dockerimagelookup.go

```diff
@@ -76,7 +76,7 @@ func (r DockerClientSearcher) Search(terms ...string) (ComponentMatches, error)
 				termMatches = append(termMatches, matches...)
 			case ErrNoMatch:
 			default:
-				return nil, err
+				glog.V(1).Infof("An error occurred searching the Docker registry: %v", err)
```

Comment:
This isn't an obvious change to me.  Why are we now ignoring any match error for docker?